# IK_command msg
# Used to send data to IK solver for solving IK
Header header
string[] frame_names
string robot_desc
float64[] cart_pose

